researcher-reports/src/components/query-item.tsx

```diff
@@ -126,6 +126,21 @@ export const QueryItem: React.FC<IProps> = (props) => {
     }
   };
 
+  const lowerQueryCompletionStatus = queryCompletionStatus.toLowerCase();
+  const running = lowerQueryCompletionStatus === "running";
+  const succeeded = lowerQueryCompletionStatus === "succeeded";
+  const failed = lowerQueryCompletionStatus === "failed";
+
+  // show the generate button until it succeeds and it is clicked
+  // the button will be disabled until it succeeds
+  const showGenerateCSVLinkButton = !succeeded || !downloadURL;
+
+  const renderCompletionStatus = () => {
+    const suffix = running ? "... (please wait)" : "";
+    const className = running ? "running" : (succeeded ? "succeeded" : (failed ? "failed" : ""));
```

Comment:
Just curious, is there a reason not to just use lowerQueryCompletionStatus as the className?

Reply:
Good point :)  There isn't really a reason now that the display code converged on that.  I'll change it - thanks!

Reply:
@lublagg fixed in https://github.com/concord-consortium/report-service/pull/198/commits/1e2eb0868355f1f815847644a96c77eef905a993